Updated documentation

NicoSchoe opened the pull request:
updated overview picture
added layer 2 configuration after device creation
minor linguistic changes
updated pictures
removed project process (not supported currently)

EvaKaul commented:
Hi @NicoSchoe ,

in general, you need to fill out a checklist for the application example based on this template:

\\\Ad001.siemens.net\dfs001\File\DE\FTH\S_SUP_PRJ-SP\TRANSFER\Anwendungsbeispiele\40_Beispiele_und_Projekte\Edge\Application_example_entry_status_template_FY2223_template_edgedocx.docx

Here are my review comments:

**Readme.md**

Used components:

-	Use latest versions for IEM and IED

Usage:

-	Here you could describe more in detail how to use the app. You could for example also transfer the chapters “Scan overview” and “Firmware update” from Installation.md file, because here it is exactly described. Or you create a new file Usage.md for these points and link to them.

**Installation.md**

Menu:
-	The menu point “Program Update” leads to nothing- no dedicated chapter

Configure Device Layer 2 access:
-	Fix typo:  “_You can configure the Layer 2 access either during the onboarding of the device through the management or **later on** directly on the edge device_.”

Configure SIMATIC Automation Tool:

-	As mentioned above, all these sub-chapters should belong to the “Usage” chapter. Are there any ‘real’ configuration points within the SAT app? Maybe you could explain the app UI and the general functions a little bit (like the file manager or the SNMP profile settings)

-	Reduce size of picture Configure_SIMATIC_Automation_Tool.PNG or better paste screenshot of the whole app entry view and highlight the “eth1” corner (top right)

NicoSchoe commented:
Hi @EvaKaul,

I just updated the Documentation. 

The checklist already exists and I just updated the repo itself. I updated the versions of the used components, created a Usage.md and made some minor chages in the language.

Can you review the changes?

EvaKaul commented:
Hi @NicoSchoe,
I just went through your changes and everything looks good!
The usage chapter is much more helpful now :+1:
From my side you can perform the merge.
